build.sh

```
@@ -1,8 +1,30 @@
+if [ -z "$version" ]; then
+    echo "Usage: $0 <version> (e.g. last 1.3.5)"
+    exit 1
+fi
+
+echo "Cleaning dist..."
+rm -rf dist
+
+echo "Building root notte package version==$version"
+uvx --from=toml-cli toml set --toml-path=pyproject.toml project.version $version
 uv build
 for package in $(ls packages); do
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)